tests/unit/states/test_mac_package.py

```diff
@@ -2,19 +2,21 @@
 
 # Import Python libs
 from __future__ import absolute_import
+import sys
 
 # Import Salt Libs
 import salt.states.mac_package as macpackage
 
 # Import Salt Testing Libs
 from tests.support.mixins import LoaderModuleMockMixin
-from tests.support.unit import TestCase
+from tests.support.unit import TestCase, skipIf
 from tests.support.mock import (
     MagicMock,
     patch
 )
 
 
+@skipIf(not sys.platform.startswith('darwin'), "Mac specific test")
```

Comment:
I don't think this is the right way to go about this. These tests have been running just fine on Linux due to mocking, so wouldn't it be better to just skip these for windows specifically?